lib/suspenders/generators/app_generator.rb

```diff
@@ -239,7 +239,7 @@ def remove_routes_comment_lines
 
     def outro
       say 'Congratulations! You just pulled our suspenders.'
-      say "Remember to run 'rails generate airbrake' with your API key."
+      say_honeybadger_outro
```

Comment:
Looking good! For consistency, let's leave the `say` call up here, and return a string from `honeybadger_outro`?